src/circle.c

```diff
@@ -370,6 +372,17 @@ spherecircle_by_center(PG_FUNCTION_ARGS)
 	PG_RETURN_POINTER(c);
 }
 
+Datum
+spherecircle_by_center_deg(PG_FUNCTION_ARGS)
+{
+	Datum res;
+	SPoint	   *p = (SPoint *) PG_GETARG_POINTER(0);
+	float8		rad = PG_GETARG_FLOAT8(1);
+	rad = deg_to_rad(rad);
```

Comment:
I would combine this line with the previous line, like so:
```
	const float8 rad = deg_to_rad(PG_GETARG_FLOAT8(1));
```
I think you can also add `const` to that variable if you do it that way? And I would call the variable `radius` to avoid confusing it with radians.

This is mainly just a coding style thing though. Feel free to ignore completely. I'm sure the compiler must optimize it the same way.

Is line 382 longer than 80 characters? I can't tell on the GitHub website. If it is, please wrap it.